### PR TITLE
doc: Updated LICENSE and technologies section

### PR DESCRIPTION
**Pull request by alanrodas**
The LICENSE stated MIT, even though we have switched to AGPL. This was updated.

The technologies section now features all the latest technologies in use, and longer and better explainations on what they do in the project. Also included is the configuration files in use for each tool, and who provides it and how.

The section about gosbtones-scripts have been added to explain how this tool hides away the configuration of some of the tooling, or how it provides defaults that should not be manually updated.

### CODE REVIEW

File: sections/technologies/technologies.md [hunk 1]
```diff
@@ -1,178 +1,492 @@
...
+We also include an `EditorConfig` file in every project.
 
-### ESLint
-In addition to `Prettier` for maintainability of codebase  throughout our project, `ESLint` statically analyzes the code for common errors, stylistic issues, and potential pitfalls, thus providing actionable feedback to fix such problems for both `Javascript` and `Typescript`, ensuring the code's quality and reliability.
 <details>
-<summary>Expand to view more about <b>ESLint</b></summary>
+<summary>Expand to view more about <b>EditorConfig</b></summary>
+Although `Prettier` is a great tool for code formatting, some IDEs do not play along with prettier, or they do not do it by default. To support more IDEs and code editors we have included an `EditorConfig` file in each project.
+
+The `EditorConfig` file, `.editorconfig` contains basic formatting configuration to tell the code editor how you require your files to be treated, uch as line endings, tab width and so on.
```
Comment (fidel-ml) [inline sections/technologies/technologies.md:294]: ", uch as line endings" -> ", such as line endings"

File: sections/technologies/technologies.md [hunk 2]
```diff
@@ -1,178 +1,492 @@
...
 
-A good starting point to know the basics on `ESLint` is [`eslint.org`](https://eslint.org/).
+This configuration should always match the one in your `Prettier` configuration, and should be the same through all projects, and as such, only should be changed through an update of `gobstones-scripts`.
+
+Again, you don't need to know much about `EditorConfig`, but it's a simple enough tool to understand by reading the provided documentation at[`editorconfig.org`](https://editorconfig.org/).
```
Comment (fidel-ml) [inline sections/technologies/technologies.md:298]: "at[" -> "at ["

File: sections/technologies/technologies.md [hunk 3]
```diff
@@ -1,178 +1,492 @@
...
+<summary>Expand to view more about <b>ESLint</b></summary>
+
+Although `Prettier` provides formatting configuration, code styling is enforced through `ESLint`. This linter provides configuration that determine things such as the order in which methods should be declared, which are valid identifiers for a variable, where you should open and close your braces and other language aware style configuration.
+
+It statically analyzes the code for common errors, stylistic issues, and potential pitfalls, thus providing actionable feedback to fix such problems, ensuring the code's quality and reliability.
+
+`ESLint` uses the concept of plugins and templates, of which we use many to enforce some particular configuration. The linter is run by default before any test, and in case of style errors, the tests cannot pass.
 
-A good starting point to know the basics on `EditorConfig` is [`editorconfig.org`](https://editorconfig.org/).
+You will find the `ESLint` configuration file, `.eslintrc` at the root of every project. Again, it should not be changed unless through updates of `gobstones-scripts`.
```
Comment (fidel-ml) [inline sections/technologies/technologies.md:324]: "unless through updates" -> "except through updates"

File: sections/technologies/technologies.md [hunk 4]
```diff
@@ -1,178 +1,492 @@
...
+
+You will usually not require changing these rules, but to understand the basic idea behind the linter, you may read the documentation at [`eslint.org`](https://eslint.org/).
```
Comment (fidel-ml) [inline sections/technologies/technologies.md:326]: "the linter, you may" -> "the linter you may"

**Review by fidel-ml — COMMENTED** [review]
Several typos have to be fixed.

File: sections/architecture/architecture.md [hunk 5]
```diff
@@ -47,11 +47,33 @@ We are currently working in redesigning some of the modules into new grouping. E
...
+
+Some configuration files needed by the tooling, such as `tsconfig.js` or `rollup.config.js`, are abstracted away by the `gobstones-script` module, which holds the configuration files. These configuration files have a default version, but, in case some module requires a particular configuration, these configuration files can be overwritten in the projects, by the subcommand `eject`.
+
+Other configuration files for some tooling, such as `.prettierc` used by `prettier`, or `.eslintrc` used by `ESLint`, are automatically created in the root of the created projects. Those files may require some sort of update as versions of the tooling are updated, or changed, and a such, the subcommand `update` allows for updating such files.
+
+Finally, the tool allows to run different actions defined through `nps`, by running the `run` subcommand.
```
[the remark this answers is not in the document]
Comment (alanrodas) [reply sections/architecture/architecture.md:60]: Explained in the technologies section for "nps".

**Pull request merged**